concent_api/utils/decorators.py

```diff
@@ -96,21 +96,21 @@ def wrapper(request, *args, **kwargs):
 def handle_errors_and_responses(view):
     """
     Decorator for handling responses from Concent
-    for golem clients
+    for golem clients and handling transactions on 'control' database.
```

Comment:
We talked about it on Slack and I said that it's fine to hard-code this decorator for the `'control'` database but now, after looking closer at the implementation, I think we should not do this. I thought the implementation was more specific to `core` but it's actually pretty generic. Enough that we could use it just fine in other apps. For example in gatekeeper. Actually I think we should start using it there and returning JSON with status 200 as simple dicts.

I think that it would be best to add a parameter for selecting the database here and always use the decorator with that parameter. Also, support the case where there's no database (parameter set to `None`).

Reply:
Ok, I will add it.